### PR TITLE
Create some battle tower SRAM labels

### PR DESCRIPTION
**Pull request by mid-kid**

### CODE REVIEW

File: engine/events/battle_tower/battle_tower.asm [hunk 1]
```diff
@@ -197,7 +199,7 @@ _BattleTowerBattle:
 	call .do_dw
 	call DelayFrame
 	ld a, [wBattleTowerBattleEnded]
-	cp $1
+	cp 1
```
Comment (Rangi42) [inline engine/events/battle_tower/battle_tower.asm]: `cp TRUE`

File: engine/battle_anims/anim_commands.asm [hunk 2]
```diff
@@ -162,7 +162,7 @@ BattleAnimRestoreHuds:
 	pop af
 	ldh [rSVBK], a
 
-	ld a, $1
+	ld a, 1
```
Comment (Rangi42) [inline engine/battle_anims/anim_commands.asm]: The various loads/reads of `hBGMapMode` should use constants eventually, so might as well leave them as hex until then.

File: engine/events/battle_tower/battle_tower.asm (hunk 1 above)
Comment (mid-kid) [reply engine/events/battle_tower/battle_tower.asm]: wInBattleTowerBattle is a weird one, it's used as bitflags (through the `bit` instruction) in a lot of places, so it's most definitely not a boolean.

Comment (Rangi42) [reply engine/events/battle_tower/battle_tower.asm]: Even more so if it's a bitflag, then please leave related values as hex until they have `*_F` constants identified.

**Pull request merged**